core/domain/prod_validation_jobs_one_off.py

```diff
@@ -200,9 +236,9 @@ def _get_external_id_relationships(cls, item):
             item: ndb.Model. Entity to validate.
 
         Returns:
-            dict(str, (ndb.Model, list(str)). A dictionary whose keys are
-            field names of the model to validate, and whose values are tuples
-            that consist of the external model class and list of keys to fetch.
+            list(ExternalModelFetcher). A list whose values are
```

Comment:
As above, perhaps reframe -- talk about what the object "is", not what is to be done to it.

Reply:
This is already talking about the list, right?